recipes-devtools/maccalc/maccalc.bb

```diff
@@ -0,0 +1,21 @@
+#
+# This file was derived from the 'Hello World!' example recipe in the
+# Yocto Project Development Manual.
+#
+SUMMARY = "MAC address manupulation utility"
+
+LICENSE = "GPL-2.0-only"
+LIC_FILES_CHKSUM = "file://main.c;md5=641b6faf83ff4caf4264dde86a5ef503"
+
+SRC_URI = "file://main.c"
+
+S = "${WORKDIR}"
+
+do_compile() {
+    ${CC} ${LDFLAGS} main.c -o maccalc
```

Comment:
I'm not sure whether bitbake defines generic CFLAGS which we should use, and whether it places them in `$CFLAGS` or embeds them in `$CC`.